### ISSUE TITLE
Problem with stacked sequential model - missing input_shapes

### ISSUE THREAD

**Issue by maciekglowka** [tensorflow/tfjs#3361]
#### TensorFlow.js version
2.0 (also earlier)
#### Browser version
FireFox
#### Describe the problem or feature request
Hi I am getting this "The first layer in a Sequential model must get an inputShape or batchInputShape argument." error.
My model does have batch_input_shape defined though. It has been converted from Python using the latest converter installed with PIP

What I have found out on this topic on stack overflow is that if the model is composed of many sequential models stacked - then each of them should have the input_shape defined.
https://stackoverflow.com/questions/61901470/tensorflowjs-says-error-the-first-layer-in-a-sequential-model-must-get-an-i

But I have so many layers and they have different input shapes..shouldn't the converter have an option to do it automatically?

Or maybe there is a solution I am not aware of ?

#### Code to reproduce the bug / link to feature request
The beginning of the model looks like so:

{"format": "layers-model", "generatedBy": "keras v2.3.0-tf", "convertedBy": "TensorFlow.js Converter v1.7.4r1", "modelTopology": {"keras_version": "2.3.0-tf", "backend": "tensorflow", "model_config": {"class_name": "Model", "config": {"name": "model_4", "layers": [{"class_name": "InputLayer", "config": {"batch_input_shape": [null, 256, 256, 1], "dtype": "float32", "sparse": false, "ragged": false, "name": "input_3"}, "name": "input_3", "inbound_nodes": []}, {"class_name": "Sequential", "config": {"name": "sequential_36", "layers": [{"class_name": "Conv2D", "config": {"name": "conv2d_26", "trainable": true, "dtype": "float32", "filters": 64, "kernel_size": [5, 5], "strides": [2, 2], "padding": "same", "data_format": "channels_last", "dilation_rate": [1, 1], "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "GlorotUniform", "config": {"seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}}, {"class_name": "LeakyReLU", "config": {"name": "leaky_re_lu_24", "trainable": true, "dtype": "float32", "alpha": 0.30000001192092896}}], "build_input_shape": [null, 256, 256, 1]}, "name": "sequential_36", "inbound_nodes": [[["input_3", 0, 0, {}]]]}, {"class_name": "Sequential", "config": {"name": "sequential_37", "layers": [{"class_name": "Conv2D", "config": {"name": "conv2d_27", "trainable": true, "dtype": "float32", "filters": 128, "kernel_size": [5, 5], "strides": [2, 2], "padding": "same", "data_format": "channels_last", "dilation_rate": [1, 1], "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "GlorotUniform", "config": {"seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}}, {"class_name": "BatchNormalization", "config": {"name": "batch_normalization_34", "trainable": true, "dtype": "float32", "axis": [3], "momentum": 0.99, "epsilon": 0.001, "center": true, "scale": true, "beta_initializer": {"class_name": "Zeros", "config": {}}, "gamma_initializer": {"class_name": "Ones", "config": {}}, "moving_mean_initializer": {"class_name": "Zeros", "config": {}}, "moving_variance_initializer": {"class_name": "Ones", "config": {}}, "beta_regularizer": null, "gamma_regularizer": null, "beta_constraint": null, "gamma_constraint": null}}, {"class_name": "LeakyReLU", "config": {"name": "leaky_re_lu_25", "trainable": true, "dtype": "float32", "alpha": 0.30000001192092896}}], "build_input_shape": [null, 128, 128, 64]}, "name": "sequential_37", "inbound_nodes": [[["sequential_36", 0, 0, {}]]]},

**Comment by rthadur** [comment]
Please check related issue [here](https://github.com/tensorflow/tfjs/issues/1613)
